tardis/plasma/properties/atomic.py

```diff
@@ -39,7 +39,7 @@ def _set_index(self, levels):
         return (levels.index, levels['energy'], levels['metastable'],
             levels['g'])
 
-class Lines(BaseAtomicDataProperty):
```

Comment:
@wkerzendorf I don't know how to fix this while leaving it as a `BaseAtomicDataProperty`. Could you recommend an alternative way of changing `set_index` so that it doesn't re-order the data?